docs/UserGuide.md

```diff
@@ -127,16 +125,30 @@ More examples:
 * The indexes **must not be duplicated**. For example, `delete 3 3` will not work.
 </div>
 
-### Importing your timetable : `import`
-
-Imports your NUSMods timetable data into your schedule.
+### Importing timetable : `import`
```

Comment:
I think the previous subheader "Importing your timetable" would be more user-centric :)